components/mdc/Select/Select.svelte

```diff
@@ -21,12 +21,17 @@ const selectedTextID = generateRandomID('select-selected-text-')
 
 let element = {}
 let mdcSelect = {}
+let previousOptionsIDsCSV = ''
```

Comment:
Will this be passed in, or how are you setting this to previous option IDs?

Reply:
It happens on line 55 inside `afterUpdate`. The first time options is populated it will be empty, meaning `optionsHaveChanged` will return true. Is that correct @forevermatt?

Reply:
@hobbitronics Yes, that's exactly it.